Makefile

```diff
@@ -33,6 +33,8 @@ endif
 # Project options are based on the app to be build (default - hello_world)
 PROJECT  ?= hello_world
 
+LINK_FOLDER ?= sw/linker
```

Comment:
should be `LINK_FOLDER ?= $(mkfile_path)/sw/linker`

Reply:
yes